### ISSUE TITLE
What's a "good" value for the loss function?

### ISSUE THREAD

**Issue by jasondoule** [experiencor/keras-yolo3#110]
I collected ~1500 labelled data and train with yolo, got a training loss of ~10, validation loss ~ 16. Obviously the fact is to use real test data to evaluate the performance, but is there a way to tell if this training loss = 10 is a "good" one? Or does it indicate I need to use more training data to see if I can push it down to 5 or even less?

Ultimately my question is, for a well-known model with a pre-defined loss function, is there a "good" standard value for the training loss?

thanks.

**Comment by ibraheemalkilanny** [comment]
I am training 25,000 images on 36 classes, getting 100+ loss after 10 Epochs !!
